### PR TITLE
[slack] Add help message for Slack users

### PR DESCRIPTION
**Pull request by Harshg999**
## What changes were proposed in this pull request?

- Help message for better usability

Screenshot 2021-05-30 at 10 30 49 PM

### CODE REVIEW

File: desktop/core/src/desktop/lib/botserver/views.py [hunk 1]
```diff
@@ -95,13 +95,62 @@ def parse_events(request, event):
...
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:link: Rich preview for Query/Gist links:* Share query/gist links in channel which unfurls in a rich preview, showing query details and result in message thread if available."
+      }
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:page_facing_up: Share directly from Editor:* Create a query gist, select channel and share your query directly from the Hue Editor window. Requires `share_from_editor` flag enabled."
```
Comment (romainr) [inline desktop/core/src/desktop/lib/botserver/views.py:132]: We should just turn it on by default

File: desktop/core/src/desktop/lib/botserver/views.py [hunk 2]
```diff
@@ -95,13 +95,62 @@ def parse_events(request, event):
...
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:eyes: Auto-detect SELECT statements:* Detects SQL SELECT statements in the channel added and suggests a gist link for improved query discussions."
+      }
+    },
+    {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:question: Surfacing Query bank:* Type `@Hue Bot queries` to explore a list of important queries from the latest query bank."
```
Comment (romainr) [inline desktop/core/src/desktop/lib/botserver/views.py:146]: No need to have 'Bot'?

**Review by romainr — APPROVED** [review]
Maybe no icons to keep it less noisy?

Also a link to the docs?

The name of each section is more or less the same as its description

File: desktop/core/src/desktop/lib/botserver/views.py (hunk 1 above)
Comment (Harshg999) [reply desktop/core/src/desktop/lib/botserver/views.py]: Will send a PR for it!

**Comment by Harshg999** [comment]
- Updated unit tests
- Updated preview below
- `much more` --> docs link
Screenshot 2021-05-31 at 1 01 31 AM

File: desktop/core/src/desktop/lib/botserver/views.py (hunk 2 above)
Comment (Harshg999) [reply desktop/core/src/desktop/lib/botserver/views.py]: FIxed and will fix in the latest app manifest too!

**Review by romainr — APPROVED** [review]
Sweet! 

Let's ship it and iterate!

**Pull request merged**